plugin-debezium/src/main/java/io/kestra/plugin/debezium/AbstractDebeziumTask.java

```diff
@@ -212,6 +214,54 @@ public AbstractDebeziumTask.Output run(RunContext runContext) throws Exception {
             .build();
     }
 
+    public Publisher<StreamOutput> stream(RunContext runContext) throws Exception {
+        // ugly hack to force use of Kestra plugins classLoader
+        Thread.currentThread().setContextClassLoader(this.getClass().getClassLoader());
+        return Flux.<StreamOutput>create(sink -> {
+                try {
+                    // restore state
+                    Path offsetFile = runContext.tempDir().resolve("offsets.dat");
+                    this.restoreState(runContext, offsetFile);
+
+                    // database history
+                    Path historyFile = runContext.tempDir().resolve("dbhistory.dat");
+                    if (this.needDatabaseHistory()) {
+                        this.restoreState(runContext, historyFile);
+                    }
+
+                    // props
+                    final Properties props = this.properties(runContext, offsetFile, historyFile);
+
+                    // callback
+                    ChangeConsumer changeConsumer = new ChangeConsumer(this, runContext, new AtomicInteger(), ZonedDateTime.now());
+
+                    // start
+                    try (DebeziumEngine<ChangeEvent<SourceRecord, SourceRecord>> engine = DebeziumEngine.create(Connect.class)
+                        .using(this.getClass().getClassLoader())
+                        .using(props)
+                        .notifying(
+                            (list, recordCommitter) -> changeConsumer.handleBatch(list, recordCommitter, sink)
+                        )
+                        .using((success, message, error) -> {
+                            if (error != null) {
+                                sink.error(error);
+                            }
+                        })
+                        .using(this.getClass().getClassLoader())
+                        .build()
+                    ) {
+                        engine.run();
+                    }
+
+                } catch (Throwable e) {
```

Comment:
```suggestion
                } catch (Exception e) {
```

It's a bad practice to catch Throwable